### PR TITLE
TotalPaymentActivity component scaffolding as per figma design

### LINKED ISSUE #8516: Create PaymentsActivityData and TotalPaymentActivity component scaffolding as per figma design
Create PaymentsActivityData and TotalPaymentActivity component scaffolding as per figma design

image

### PR DESCRIPTION
**Pull request by naman03malhotra**
Fixes #8516 

#### Changes proposed in this Pull Request
Created `<TotalPaymentActivity />` component scaffolding as per figma design

image

#### Testing instructions

- Cross-check with Figma design
- Make sure tests don't break

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `8517` or branch name `feat/8516-total-payment-activity` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=feat/8516-total-payment-activity) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 9f59fee84d60bc1c02895b6dd93d9e75782769f1
- Build time: 2024-04-03 06:08:31 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/components/payment-activity/other-payments-data.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+const OtherPaymentsData: React.FC = () => {
```
Comment (jessy-p) [inline client/components/payment-activity/other-payments-data.tsx]: Did not prefer the name `OtherPaymentsData`. Maybe something like `DataHighlights`as it is referred to in the design?

Comment (jessy-p) [reply client/components/payment-activity/other-payments-data.tsx]: Changed

File: client/components/payment-activity/style.scss [hunk 2]
```diff
@@ -1,15 +1,55 @@
...
+		}
+	}
+}
+
+.payments-activity-data {
+	display: flex;
+	padding: 24px;
+	border-top: 1px solid $gray-200;
+	border-bottom: 1px solid $gray-200;
+
```
Comment (Jinksi) [inline client/components/payment-activity/style.scss]: ⛏️ Nit: we should remove these border-top/bottom CSS declarations, as they are duplicating borders that are handled by CardHeader and CardFooter (Emoji Survey).
```suggestion
```

**Review by Jinksi — APPROVED** [review]
LGTM @jessy-p & @naman03malhotra 🤙

I've tested with the feature flag on:

(screenshot after suggested CSS change to remove border-top/bottom)

image

With feature flag off, the Payment activity flag remains hidden.

File: client/components/payment-activity/style.scss (hunk 2 above)
Comment (nagpai) [inline client/components/payment-activity/style.scss:23]: We will need to add a breakpoint for mobile view. 

```suggestion
@include breakpoint( '<660px' ) {
		flex-direction: column;
	}

```

#### Before change: 

image


#### After change:

image

**Review by nagpai — CHANGES_REQUESTED** [review]
Noticed a small SCSS change needed to accomodate mobile view.

File: client/components/payment-activity/style.scss (hunk 2 above)
Comment (jessy-p) [reply client/components/payment-activity/style.scss:23]: Thanks @nagpai. Implemented!

Comment (jessy-p) [reply client/components/payment-activity/style.scss]: Thanks @Jinksi! Done.

**Review by jessy-p — APPROVED** [review]

File: client/components/payment-activity/style.scss (hunk 2 above)
Comment (nagpai) [reply client/components/payment-activity/style.scss:23]: This looks great now, Thank you!

**Review by nagpai — APPROVED** [review]
Looks great to 🚢 . My last feedback was addressed!

**Pull request merged**